### PR TITLE
Add documentation for TizenNUIGadget RPK development in VS

### PR DESCRIPTION
**Pull request by adhavan-m**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by sangwook96 — APPROVED** [review]
Great and Thank you very much!!

File: docs/application/vstools/Tizen/nuigadget-rpk.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+To create a Tizen .NET NUIGadget project in Visual Studio:
+
+1. Launch the Visual Studio tool.
+
+2. In the Visual Studio menu, select **File &gt; New &gt; Project**.
+
+   ![Create Tizen project](media/vs2022_nuigadget_create_1.png)
+
+3. In New project menu, select C# and Tizen from the dropdown list. Then, select **Tizen DotNet project** and click **Next**.
```
Comment (safir-srbd) [inline docs/application/vstools/Tizen/nuigadget-rpk.md:19]:
```suggestion
3. In the new project menu, select C# and Tizen from the dropdown list. Then, select **Tizen DotNet Project** and click **Next**.
```

File: docs/application/vstools/Tizen/nuigadget-rpk.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# TizenNUIGadget RPK development
+
+
+## Develop application
+
+The following sections explain how to use Visual Studio Extension for creating Tizen .NET NUIGadget project and building it as RPK package.
```
Comment (safir-srbd) [inline docs/application/vstools/Tizen/nuigadget-rpk.md:6]:
```suggestion
The following sections explain how to use Visual Studio Extension for creating Tizen .NET NUIGadget project and building it as an RPK package.
```

File: docs/application/vstools/Tizen/nuigadget-rpk.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+   ![Visual Studio screen](media/vs2022_nuigadget_create_5.png)
+
+
+### Build your project and package it as RPK
+
+1. To build TizenNUIGadget and package it as RPK, select **Build Solution** in the **Solution Explorer** window.
+
+   ![Build project](media/vs2022_nuigadget_build_1.png)
+
+2. After build is successful, RPK package location will be shown in Output logs.
```
Comment (safir-srbd) [inline docs/application/vstools/Tizen/nuigadget-rpk.md:42]:
```suggestion
2. After the build is successful, the RPK package location will be shown in the Output logs.
```

**Review by safir-srbd — COMMENTED** [review]
@adhavan-m Hello. Hope you are well. I have gone through the PR, and have provided some suggestions below. Please incorporate them.

**Comment by adhavan-m** [comment]
@safir-srbd I have incorporated the suggestions. Please have a look

File: docs/application/vstools/Tizen/nuigadget-rpk.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+To create a Tizen .NET NUIGadget project in Visual Studio:
+
+1. Launch the Visual Studio tool.
+
+2. In the Visual Studio menu, select **File &gt; New &gt; Project**.
+
+   ![Create Tizen project](media/vs2022_nuigadget_create_1.png)
+
+3. In the new project menu, select C# and Tizen from the dropdown list. Then, select **Tizen DotNet project** and click **Next**.
```
Comment (safir-srbd) [inline docs/application/vstools/Tizen/nuigadget-rpk.md:19]:
```suggestion
3. In the new project menu, select C# and Tizen from the dropdown list. Then, select **Tizen DotNet Project** and click **Next**.
```

**Review by safir-srbd — COMMENTED** [review]
@adhavan-m Thank you for implementing my suggestions. However, it seems like you missed out on one small change. Please incorporate the suggested change.

**Review by safir-srbd — APPROVED** [review]
@adhavan-m Thank you for making the change. Everything seems to be in order now. I am approving this PR for merging.

**Pull request merged**